app/controllers/planning_applications/press_notices_controller.rb

```diff
@@ -28,16 +51,18 @@ def create
       end
     end
 
-    alias_method :update, :create
-
     private
 
     def press_notice_params
       params.require(:press_notice).permit(:required, :other_reason, reasons: [])
     end
 
+    def build_press_notice
+      @press_notice = @planning_application.press_notices.new
+    end
+
```

Comment:
If this is being moved out of `PlanningApplication` can it be deleted there too? Or do we need two different versions of it?